README.md

```diff
@@ -101,7 +102,8 @@ Example commands:
 
```

Comment:
![image](https://github.com/BHObserver/To-Do-List/assets/93039359/b8dc4a07-270c-4cd3-9347-c82e03082443)

**[OPTIONAL]**

- [ ] It would be nice if you add your message after the Acknowledgement heading to make your readme consistent and professional 👍